modules/letsencrypt/templates/nginx.conf.erb

```diff
@@ -0,0 +1,30 @@
+server {
+	# We can only set backlog once per port (so this will be applied to the others below)
+	listen 80 deferred backlog=4096;
+	listen [::]:80 deferred ipv6only=on backlog=4096;
+	listen 443 ssl deferred http2 backlog=4096;
+	listen [::]:443 deferred ssl http2 ipv6only=on backlog=4096;
+
+	server_name miraheze.org www.miraheze.org;
+	root /srv/mediawiki/landing;
```

Comment:
Make sure landing is be cloned then, and this would also mean /srv/mediawiki would need to exist. Landing is currently done with deploy-mediawiki for MediaWiki, but for this just could be cloned probably.

Reply:
Do not load landing on puppet111! Also you may need to copy how we do domains in mw.

Reply:
Yes, I wasn't saying that landing had to be cloned, which seemed to be a bit of a misunderstanding on my part here. I meant that if the web root is set to landing we need landing, but the ideal way would be to set the root to /var/www/challenges or something.